### PR TITLE
Show when a VCF has comments

### LINKED ISSUE #398: Indicate whether comments are present on the runs page
A good analogy for this is the GitHub Issues page:

screen shot 2014-12-15 at 3 14 28 pm

### PR DESCRIPTION
**Pull request by tavinathanson**
I think it's fairly self-explanatory! Fixes #398 

[Review on Reviewable](https://reviewable.io/reviews/hammerlab/cycledash/422)

### CODE REVIEW

**Comment by tavinathanson** [comment]
@ihodes Oops, realized I didn't totally finish proofreading the SQL query, I'll let you know when it's ready

File: cycledash/static/css/runs.css [hunk 1]
```diff
@@ -42,6 +42,19 @@ td span.run-id {
...
   font-size: 0.92em;
 }
+span.comment-bubble {
+  background: url(/static/img/speech-bubble-noun_5483.svg) no-repeat 0px 3px;
+  background-size: 15px 15px;
+  width: 15px;
+  height: 15px;
+  display: inline-block;
+}
+span.no-comment-dimmed {
```
Comment (ihodes) [inline cycledash/static/css/runs.css]: I'd drop the `-dimmed`, because that isn't semantically relevant

File: cycledash/templates/runs.html [hunk 2]
```diff
@@ -77,9 +78,16 @@
           {%- if run['tumor_bam_uri'] %}<span class="label label-info" title="Has an associated tumor BAM">tumor</span>{% endif -%}
           {%- if run['normal_bam_uri'] %}<span class="label label-info" title="Has an associated normal BAM">normal</span>{% endif -%}
         </td>
+        <td class="comments">
+          <span class="comment-bubble {{ '' if run['num_comments'] > 0 else 'no-comment-dimmed'}}">
```
Comment (ihodes) [inline cycledash/templates/runs.html]: You don't need the `> 0` since 0 is falsey… you may know this and chose to do it anyway.  Just FYI.

**Comment by ihodes** [comment]
LGTM % the CSS classname

File: cycledash/templates/runs.html (hunk 2 above)
Comment (tavinathanson) [inline cycledash/templates/runs.html]: Nah, I had just forgotten Python's falsey values. I'll get rid of it. (Even though I can see both arguments for keeping it and getting rid of it.)

Comment (ihodes) [inline cycledash/templates/runs.html]: Word. Yeah I'd lean towards not doing the explicit check, but it really depends on the day. I believe taking advantage of `0` being falsey is Pythonic, but I'm not a huge fan of that explanation for justifying how I write code…

Comment (tavinathanson) [inline cycledash/templates/runs.html]: :+1:

**Pull request merged**